src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
-body {
+@import "./utils/variables";
```

Comment:
```suggestion
@import "./utils/variables";
```
If you have this file here, why do you need it in __product_card.scss?

Reply:
i had troubles with variables and this fixed them((